### PR TITLE
Classes Roster BackOffice and FieldStaff eingefügt

### PR DESCRIPTION
**Pull request by deutschha**
Signed-off-by: Harald Deutsch <haralddeutsch@gmail.com>

### CODE REVIEW

File: src/main/java/model/schedules/BackOffice.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import java.util.ArrayList;
+
+public class BackOffice extends Roster{
+
+	public BackOffice(int diffHours, int diffVacation, ArrayList tasks, ArrayList openTrainings, ArrayList timetable) {
+		super(diffHours, diffVacation, tasks, openTrainings, timetable);
+		// TODO Auto-generated constructor stub
+	}
+	
+}
```
Comment (BeWie24) [inline src/main/java/model/schedules/BackOffice.java:12]: good job!

File: src/main/java/controller/schedules/Roster.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+import java.util.ArrayList;
+//Roster  beudeutet Dienstplan
+public class Roster {
+	private int diffHours, diffVacation;
+	protected ArrayList tasks, openTrainings;
+	protected ArrayList timetable;
+	
+	public Roster(int diffHours, int diffVacation, ArrayList tasks, ArrayList openTrainings, ArrayList timetable) {
+		super();
```
Comment (ghost) [inline src/main/java/controller/schedules/Roster.java]: super() call could be removed?

File: src/main/java/model/schedules/BackOffice.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package model.schedules;
+
+import java.util.ArrayList;
+
+public class BackOffice extends Roster{
+
+	public BackOffice(int diffHours, int diffVacation, ArrayList tasks, ArrayList openTrainings, ArrayList timetable) {
+		super(diffHours, diffVacation, tasks, openTrainings, timetable);
+		// TODO Auto-generated constructor stub
```
Comment (eliritt) [inline src/main/java/model/schedules/BackOffice.java]: delete TODO

File: src/main/java/model/schedules/BackOffice.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package model.schedules;
+
+import java.util.ArrayList;
+
+public class BackOffice extends Roster{
```
Comment (ghost) [inline src/main/java/model/schedules/BackOffice.java]: Suggest rename of class to BackOfficeRoster so everyone knows that the Roster is meant?

File: src/main/java/model/schedules/FieldStaff.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
+package model.schedules;
+
+import java.util.ArrayList;
+
+
+public class FieldStaff extends Roster {
```
Comment (ghost) [inline src/main/java/model/schedules/FieldStaff.java]: Suggest rename of class to FieldStaffRoster (reason see above comment)

File: src/main/java/model/schedules/FieldStaff.java [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+
+public class FieldStaff extends Roster {
+	
+	public String partner;
+	public String car;
+	public String area;
+	
+	
+	public FieldStaff(int diffHours, int diffVacation, ArrayList tasks, ArrayList openTrainings, ArrayList timetable, String Partner, String car, String area) {
```
Comment (eliritt) [inline src/main/java/model/schedules/FieldStaff.java]: constructor: partner must be written in lower case

**Review by eliritt — COMMENTED** [review]
Please consider my comments :)

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request closed without merging**